Sources/Core/Model/Channel/Channel+Events.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Channel+Events.swift
+//  StreamChatCore
+//
+//  Created by Alexey Bukhtin on 10/01/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public extension Channel {
+    
+    /// Observe channel events.
+    /// - Parameters:
+    ///   - eventType: an event type.
+    ///   - completion: a completion block with `Event`.
+    /// - Returns: a subscription.
+    func onEvent(_ eventType: EventType, _ completion: @escaping ClientCompletion<Event>) -> Subscription {
+        return rx.onEvent(eventType).bind(to: completion)
```

Comment:
`bindOnce` instead of `bind` (to all functions in this file)?

Reply:
Nope. You are observing events continuously.